DeDriftAndResample/DeDriftAndResamplePipeline.sh

```diff
@@ -27,1094 +27,614 @@
-       timeseries, only required when using --multirun-fix-names.
-  [--multirun-fix-extract-names=<day1run1@day1run2%day2run1@day2run2>] @ and % delimited list of lists of
-       fMRIName strings to extract, one list for each multi-run ICA+FIX group in --multirun-fix-names (use
-       a NONE instead of the group's runs and a NONE in --multirun-fix-extract-concat-names to skip this
-       for a group), only required when using --multirun-fix-extract-concat-names.  Exists to enable
-       extraction of a subset of the runs in a multi-run ICA+FIX group into a new concatenated series
-       (which is then named using --multirun-fix-extract-concat-names).
-  [--multirun-fix-extract-concat-names=<day1_newconcat@day2_newconcat>] @-delimited list of names for the
-       concatenated extracted timeseries, one for each multi-run ICA+FIX group (i.e. name in
-       --multirun-fix-concat-names; use NONE to skip a group).
-  [--multirun-fix-extract-extra-regnames=<regname@regname>] extract MR FIX runs for additional surface
-       registrations, e.g. --multirun-fix-extract-extra-regnames=MSMSulc
-  [--multirun-fix-extract-volume={TRUE, FALSE}] whether to also extract the specified MR FIX runs from the
-       volume data, requires --multirun-fix-extract-concat-names to work, default FALSE.
-  [--fix-names=<ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that will
-       have single-run ICA+FIX reapplied to them (could be either rfMRI or tfMRI). Do not specify runs
-       processed with MR FIX here. Previously known as --rfmri-names.
-  [--dont-fix-names=<not@ICA+FIXed@fMRI@Names>] @-delimited fMRIName strings corresponding to maps that
-       will not have ICA+FIX reapplied to them (not recommended, MR FIX or at least single-run ICA+FIX is
-       recommended for all fMRI data). Previously known as --tfmri-names.
-   --smoothing-fwhm=<number> Smoothing FWHM that matches what was used in the fMRISurface pipeline
-   --high-pass=<high-pass filter used in ICA+FIX>
-   --motion-regression={TRUE, FALSE} whether FIX should do motion regression
-  [--myelin-target-file=<path/to/myelin/target/file>] A myelin target file is required to run this 
-       pipeline when using a different mesh resolution than the original MSMAll registration.
-  [--input-reg-name=<string>] A string to enable multiple fMRI resolutions (e.g._1.6mm)
-  [--matlab-run-mode={0, 1, 2}] defaults to ${G_DEFAULT_MATLAB_RUN_MODE}
-       0 = Use compiled MATLAB
-       1 = Use interpreted MATLAB
-       2 = Use interpreted Octave
-
-EOF
-}
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
+g_matlab_default_mode=1
+
+#description to use in usage - syntax of parameters is now explained automatically
+opts_SetScriptDescription "implements DeDrift and Resample Pipeline"
```

Comment:
This is the dedrift and resample pipeline, it is a little odd to say that it implements itself.  You could just put "De-Drift and Resample" to match the previous help info.  Or, we could make it more informative, and say "resamples many types of data after a surface registration".  The "dedrift" part is basically an unfortunate red herring, made more unfortunate by legacy considerations requiring the use of the dedrifting option for MSMAll.